positional/tests/name_clashes.rs

```diff
@@ -0,0 +1,56 @@
+// Force name clashes to make sure the macros generate code with fully qualified imports.
+
+pub trait ToPositionalField {}
+pub trait ToPositionalRow {}
+pub trait FromPositionalRow {}
+
+mod positional {}
+
+#[derive(::positional::ToPositionalRow, ::positional::FromPositionalRow, PartialEq, Debug)]
+struct Data {
+    #[field(size = 5)]
+    name: String,
+    #[field(size = 5, filler = '-', align = "right")]
+    age: i32,
+    #[field(size = 20)]
+    address: String,
+}
+
+impl Data {
+    #[allow(dead_code)]
+    fn from_positional_row(_: &str) -> Self {
+        unreachable!()
+    }
+
+    #[allow(dead_code)]
+    fn to_positional_row(self) -> String {
+        unreachable!()
+    }
+}
```

Comment:
uh! I didn't think about that :exploding_head:

Reply:
This is probably overkill; I just wanted to make sure that the generated code definitely calls the versions in the Trait impls generated by positioal :D